runtime/ggma/src/Macro.h

```diff
@@ -17,20 +17,11 @@
 #include <cstdlib>
 
 /**
- * @file  ggma_macro.h
+ * @file  Macro.h
  * @brief Common macros for GGMA error handling and utilities
  */
-#ifndef __GGMA_GGMA_MACRO_H__
-#define __GGMA_GGMA_MACRO_H__
-
-#define GGMA_ENSURE(a)               \
-  do                                 \
-  {                                  \
-    if ((a) != GGMA_STATUS_NO_ERROR) \
-    {                                \
-      exit(-1);                      \
-    }                                \
-  } while (0)
```

Comment:
It will be under `ggma_run.cc`.
Then, it will be in `tests/tools/libs` for two or more tools.
Then, it will be in public header if other use cases occurrs.